<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="txt1">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>15</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>제형</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box1">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>50</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>전체</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>필름코팅정</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>나정</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경질캡슐제|산제</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>연질캡슐제|현탁상</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>연질캡슐제|액상 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>기타</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="box2">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>50</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>전체</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>원형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>장방형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>타원형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>반원형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>삼각형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>사각형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>마름모형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>오각형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>육각형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>팔각형</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>기타</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="box3">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>50</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>전체</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>하양</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>노랑</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>주황</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>분홍</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>빨강</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>갈색</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>연두</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>초록</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>청록</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>파랑</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>남색</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>자주</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>보라</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>회색</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>검정</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>투명</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="box4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>전체</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>없음</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>기타</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="txt2">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>15</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>모양</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt3">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>15</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>색상</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>15</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>분할선</string>
   </property>
  </widget>
  <widget class="QLabel" name="MainTxt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>220</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="smalltxt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 87, 83);</string>
   </property>
   <property name="text">
    <string>알약의 모양, 색상, 마크, 문자 등을 입력하여 제품명을 검색할 수 있습니다. </string>
   </property>
  </widget>
  <widget class="QPushButton" name="Fbtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Going
to find</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>480</width>
     <height>80</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>478</width>
      <height>78</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="medlist">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>480</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AppleSDGothicNeoUL00</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>210</y>
     <width>220</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="medinfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>220</width>
     <height>104</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoT00</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="openLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextbtn">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>80</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="undobtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="medlist2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AppleSDGothicNeoUL00</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>전체</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="savebnt">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>270</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="medImg">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>99</y>
     <width>250</width>
     <height>163</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>100</y>
     <width>3</width>
     <height>195</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <zorder>medImg</zorder>
  <zorder>txt1</zorder>
  <zorder>box1</zorder>
  <zorder>box2</zorder>
  <zorder>box3</zorder>
  <zorder>box4</zorder>
  <zorder>txt2</zorder>
  <zorder>txt3</zorder>
  <zorder>txt4</zorder>
  <zorder>MainTxt</zorder>
  <zorder>line</zorder>
  <zorder>smalltxt</zorder>
  <zorder>Fbtn</zorder>
  <zorder>scrollArea</zorder>
  <zorder>line_2</zorder>
  <zorder>medinfo</zorder>
  <zorder>nextbtn</zorder>
  <zorder>undobtn</zorder>
  <zorder>medlist2</zorder>
  <zorder>savebnt</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
